app/webhook_test.go

```diff
@@ -753,7 +753,7 @@ func TestDoOutgoingWebhookRequest(t *testing.T) {
 		server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			io.Copy(w, strings.NewReader(`{"text": "Hello, World!"}`))
 			w.Header().Set("Content-Type", "application/json")
-			// to test getting the correct digest in header, find and uncomment (nowStamp = "1576740798") in app/webhook.go
+			// to test getting the correct digest in header, find and uncomment (now = "1576740798") in app/webhook.go
```

Comment:
Instead of disabling the test, could we write the necessary code to validate the signature just like a Go server might have to do when using this feature in production? How would such a server know the timestamp to use and by which to verify the signature?

Reply:
done